mix.exs

```diff
@@ -43,10 +43,10 @@ defmodule Bugsnex.Mixfile do
   # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:httpoison, "~> 0.8.3"},
+      {:httpoison, "~> 0.11"},
       {:poison, "~> 1.5 or ~> 2.0"}, # to be compatible with phoenix_ecto
```

Comment:
here we'd need a new version - or well we also want to allow poision 3.x so I guess `>= 1.5` might be finde